routes/web.php

```diff
@@ -1,8 +1,28 @@
 <?php
 
+use App\Http\Controllers\DashboardController;
+use App\Http\Controllers\FaqController;
+use App\Http\Controllers\PostController;
+use App\Http\Controllers\StaticContentController;
 use Illuminate\Support\Facades\Route;
 
-Route::get('/', function () {
-    return view('welcome');
-})->name('home');
+Route::get('/', [StaticContentController::class, 'index'])->name('home');
+Route::get('/profile', [StaticContentController::class, 'profile'])->name('profile');
 
+Route::get('/blog', [PostController::class, 'index'])->name('posts.index');
```

Comment:
It is possible to define all 7 RESTful resource in just one command: `Route::resource(...)`. Saves development time.